spec/pronounce_spec.rb

```diff
@@ -19,7 +19,7 @@
     end
 
     it 'groups the phones by syllable' do
-      expect(Pronounce.how_do_i_pronounce('monkeys')).to eq [%w[M AH1 NG], %w[K IY0 Z]]
+      expect(Pronounce.how_do_i_pronounce('monkeys')).to eq [%w(M AH1 NG), %w(K IY0 Z)]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.